modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/SynonymTokenFilterFactory.java

```diff
@@ -50,6 +51,7 @@
     private final boolean lenient;
     protected final Settings settings;
     protected final Environment environment;
+    private final boolean updateable;
```

Comment:
Since this is a noop for now I think it would be better to test the functionality in a custom token filter only used in test. We can work on synonyms update as a follow up but we should not add this option if the logic is not implemented ?

Reply:
Good point, will see how that would look like and what that means for testing.